### DISCUSSION TITLE
Need Advice on HDMI to Component adapter

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#121] [General]
I am going to have to change my cable boxes and move to IPTV, Rogers Ignite here in Canada which is essentially a rebranding of Comcast Xfinity.  I bought this on Amazon (https://www.amazon.ca/dp/B09N8K7SPQ) but the quality is poor.  The picture is dark and blurry, much worse than the direct Component feed.

Anyone have a similar device that they are happy with?

Or should I go to something like a uray HDMI encoder.  They are a lot more expensive.

**Comment by TGsell1** [comment]
The URay boxes give terrific quality I have two getting fed from Windows Media Center/ Cable card they do no need HDMI HDCP stripping and work well with OpenDCT.   Sent from my iPhoneOn Mar 23, 2024, at 12:22 PM, wayner9 ***@***.***> wrote:﻿
I am going to have to change my cable boxes and move to IPTV, Rogers Ignite here in Canada which is essentially a rebranding of Comcast Xfinity.  I bought this on Amazon (https://www.amazon.ca/dp/B09N8K7SPQ) but the quality is poor.  The picture is dark and blurry, much worse than the direct Component feed.
Anyone have a similar device that they are happy with?
Or should I go to something like a uray HDMI encoder.  They are a lot more expensive.

**Reply by wayner9** [reply]
Thanks, that is my other option.  Does this one on AliExpress seem like yours?  https://www.aliexpress.com/item/32826096725.html

**Reply by emveepee** [reply]
I have the URay (I got the one with HDMI out) I consider it darker then the HDPVR so not terrific IMO.  Once I get the HDPVR2 working on Linux in the other thread on blasting I will upload a video showing both.  HDPVR2 are pretty cheap second hand.

**Reply by wayner9** [reply]
Do you need a HDCP stripper with the HDPVR2?  I am not sure if they work in unRAID which could be an issue for me.

It looks like they are about $80-100 including shipping to Canada.

**Reply by emveepee** [reply]
Yes, I figure the Hauppauge firmware supports HDCP and I didn't notice a difference with windows.

**Reply by wayner9** [reply]
Since Hauppauge supports HDCP you do need to strip it?  If so, what do you use?

**Reply by emveepee** [reply]
I did write yes earlier.  I used a device from Amazon I got many years ago as a splitter, but I am not going to recommend anything since the specs on the generic devices can change purchase to purchase.  Anyway you have diverged from your original topic (and now you have 3 threads going here on this same issue) so I will stick to the thread on XiOne with my personal approach to make responding simpler from me and maybe there will be some more on topic posts in this one.

**Comment by TGsell1** [comment]
My Uray encoded output is via IP maybe that makes a difference anyway I can adjust some of the image qualities from the WMC video outputs.   Sent from my iPhoneOn Mar 23, 2024, at 6:28 PM, emveepee ***@***.***> wrote:﻿
I have the URay (I got the one with HDMI out) I consider it darker then the HDPVR so not terrific IMO.  Once I get the HDPVR2 working on Linux in the other thread on blasting I will upload a video showing both.  HDPVR2 are pretty cheap second hand.

**Reply by wayner9** [reply]
I really like network encoders rather than devices that are more dependent on a PC and PCI slots, or USB and the required drivers.  That's why I moved from Hauppauge to HDHR for OTA encoding a decade or so ago.  I would like to do that with cable/IPTV capture but I don't know if it is worth trying to get over the hurdles with so little support these days, both in terms of other users, and online resources, like the old forums, being hard to find/navigate.  Both for the capture side of the equation and the channel changing.

**Comment by emveepee** [comment]
Here's a couple of samples, one from the HDPVR2 and one from the URay on Linux https://filetransfer.io/data-package/S9N8utDa#link with sha256 sum of 745b245634b6c89475bf31e9c3ce8c50ac77befcc5bc8d8dce028d5d60a0331b  different boxes but the same Rogers service.

Although it is easy to tell from the file spec there probably isn't that much of a difference as I thought.  Note I don't use 60 fps for the uray since I can't always stream cleanly.  The HDPVR is adaptive.

**Comment by TGsell1** [comment]
File #2 looks a little better in my opinion but both close and pretty good.

**Comment by wayner9** [comment]
I have tried a second adapter and although it looks like it is cheaper quality it seems to provide a better quality picture.  This is the second one:  https://www.amazon.ca/dp/B07PMGBVV8?ref=ppx_yo2ov_dt_b_product_details&th=1
![image](https://github.com/OpenSageTV/Community/assets/10366145/7008e95c-b234-4935-bfc5-3d40f8e3b600)

**Reply by Swagger200** [reply]
I am in the same situation you - a Rogers customer being forced on to Ignite, and losing access to a component out which allows use of HD-PVR devices. I have some experience with cheap Amazon HDMI to component converters and have had some success, but only if the source is not using HDCP at all, or is using HDCP up to version 1.2 .  It would be good to know what level of HDCP the Ignite boxes are using, I would suspect that they are higher than 1.2 as that version of HDCP is quite old now.

**Reply by Swagger200** [reply]
From some Googling it appears the Comcast boxes that Rogers is using require HDCP 2.2

**Reply by emveepee** [reply]
The definitely do not need HDCP 2.2  If you think about it restricting service to only the newest most powerful televisions would not make business sense  I am pretty sure I have used 1.2 cables for capturing

**Reply by wayner9** [reply]
I have tried two cheapo Amazon adapters and both of them worked perfectly well with IgniteTV boxes so I have yet to experience problems caused by HDCP.

**Comment by seansd** [comment]
@wayner9 FWIW I have been using the URay devices for years and upping the bitrate / tweaking the settings works pretty darn well for me as far as I'm concerned.  And honestly I don't think I have had lock up issues in years since I moved away from the HDPVR boxes.

**Reply by wayner9** [reply]
Thanks Sean - are you happy with the picture quality?  I have not had issues with HDPVR boxes and I have been using them for about 15 years.  But my new cable box doesn't have component out so I need component->HDMI adapters and I am not totally happy with the ones that I have.  And I like have network devices like the Uray rather than devices attached to the PC's USB device as they tend to be less finicky around stuff like drivers.

What do you do to use the Uray devices as a tuner in Sage?  Do you use OpenDCT?  How do you configure it - would you have a link to the old forums or some other document?

**Comment by seansd** [comment]
Yes I do use OpenDCT.  Sadly I don't have a link handy since the forums are archived and I no longer have my handy dandy user control panel. But here's the archive link that's probably the most useful: https://forums.sagetv.com/forums/showthread_t_62774.html?t=62774
I'm still using serial tuning for my DirecTV boxes.

As far as I'm concerned the picture quality is definitely above average.  I'm capturing straight from HDMI (Uray's don't care), and I set the STBs to stay at 1080i.  I would go up to 1080p, but from what I remember DirecTV is pretty much always a 720p source anyway, so I just stick with that and up the bitrate and really don't notice much if any loss at all.  The only downside I have is it's strictly 2 channel stereo for audio, but that's not a deal breaker for me.

I don't see linear cable / sat ever doing 4K or above since it wouldn't be worth their time any more. . . but maybe broadcast sources might hit that someday.l

**Reply by wayner9** [reply]
"I don't see linear cable / sat ever doing 4K or above since it wouldn't be worth their time any more. . . but maybe broadcast sources might hit that someday." 

Here in Canada we have had 4K sports channels for many years.  All Toronto Blue Jays home sames since the 2017 season have been in HD on one of the local cable Sports channels.  Toronto Maple Leaf and Toronto Raptors home games have also been in HD since the same time. 

So Rogers Cable carries three HD sports channels - TSN4K, Sportsnet4K and Sportsnet1 4K.

**Comment by seansd** [comment]
Honestly if I could pay the money to have the old forum restored, I would do that. . . i mean seriously how much can that cost?

**Reply by Sagetvmaster** [reply]
I think most of us would be happy to pay a bit for the old forum to be reborn, but I don't think Narflex wants to support that anymore?  Or maybe it's Google who's in the way, but yeah, it's sorely missed...
